### PR TITLE
Exclude gnu.org and fsf.org from link checker

### PR DESCRIPTION
**Pull request by samcunliffe**
The link checker is timing out at gnu.org and fsf.org. Propose to exclude as this is not the first time we've encountered this. And we can't _remove_ the links: it's part of the licenses.

Would unblock @renovate bot's many updates.

### CODE REVIEW

File: .github/workflows/links.yml [hunk 1]
```diff
@@ -23,11 +23,14 @@ jobs:
...
-          # the whole of the project slug README since the links are
+          # gnu.org and fsf.org because they're often down but are still the right
+          # websites to link to!
+          #
+          # Ignore the whole of the project slug README since the links are
           # necessarily not real links - i.e. demos and to be filled by
           # cookiecutter values.
           args:
-            "--verbose --exclude https://forum.image.sc/ --exclude docs/pages
+            "--verbose --exclude https://forum.image.sc/ --exclude http://www.gnu.org --exclude http://fsf.org --exclude docs/pages
```
Comment (paddyroddy) [inline .github/workflows/links.yml:33]:
```suggestion
            "--verbose --exclude https://forum.image.sc/ --exclude
            http://www.gnu.org --exclude http://fsf.org --exclude docs/pages
```

**Review by paddyroddy — APPROVED** [review]
One small suggestion

**Comment by paddyroddy** [comment]
@samcunliffe stop automerging like that! Otherwise, what is the point in reviewing?

**Comment by samcunliffe** [comment]
> @samcunliffe stop automerging like that! Otherwise, what is the point in reviewing?

🤖 the automerge robot and our linters are happy with my linebreaks.

**Comment by paddyroddy** [comment]
> 🤖 the automerge robot and our linters are happy with my linebreaks.

That's because you wouldn't allow https://prettier.io/docs/en/options.html#prose-wrap

**Comment by samcunliffe** [comment]
Did I? That doesn't sound like a think I would block...

We had it in 
- https://github.com/UCL-ARC/python-tooling/pull/356

**Comment by samcunliffe** [comment]
When did it disappear?

**Comment by samcunliffe** [comment]
> > 🤖 the automerge robot and our linters are happy with my linebreaks.
> 
> That's because you wouldn't allow https://prettier.io/docs/en/options.html#prose-wrap

Ha! Found it. @paddyroddy, I'm sorry to report that it was _you_ that removed that:
- https://github.com/UCL-ARC/python-tooling/pull/364

**Comment by paddyroddy** [comment]
> Ha! Found it. @paddyroddy, I'm sorry to report that it was _you_ that removed that:
> 
>     * [Fix #361 #364](https://github.com/UCL-ARC/python-tooling/pull/364)

I'm fully aware, it was due to your markdown linking

**Pull request merged**